core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.ts

```diff
@@ -45,17 +45,14 @@ export class OpportunitiesListComponent {
   @Input() opportunityHeadingTruncationLength: number;
   @Input() opportunityType: string;
 
-  // The initial last page number before we've loaded the first opportunities.
-  private static readonly INITIAL_LAST_PAGE_NUMBER = 1000;
-
   loadingOpportunityData: boolean = true;
-  lastPageNumber: number = OpportunitiesListComponent.INITIAL_LAST_PAGE_NUMBER;
   opportunities: ExplorationOpportunity[] = [];
   visibleOpportunities = [];
   directiveSubscriptions = new Subscription();
   activePageNumber: number = 1;
   OPPORTUNITIES_PAGE_SIZE = constants.OPPORTUNITIES_PAGE_SIZE;
   more: boolean = false;
+  isOnLastPage: boolean = true;
```

Comment:
Could you please change this to userIsOnLastPage, so that it doesn't start with a verb? Otherwise, it has function semantics.

Then you can rename calculateIsOnLastPage() to isUserOnLastPage().

Reply:
Done.